submissions/rubyAdeniregun/algorithmsAndDataStructures/arrays.js

```diff
@@ -0,0 +1,49 @@
+var products = [
+    'CanonKitLens15-55mm', 
+    'Nikon3100',
+    'SamsungNX300',
+    'CanonEOS80D',
+    'SonyAlphaA7s',
+    'SonyNX300M',
+    'PanasonicLumixGH4',
+    'PanasonicLumixDC-GH5',
+    'XiaomiY1-M1'
+];
+
+var product_price = [
+    '500.00', 
+    '200.00',
+    '750.00',
+    '2000.00',
+    '1600.00',
+    '300.00',
+    '800.00',
+    '5000.00',
+    '500.00'
+];
+
+/*to check the length of the product names */
+
+for (i=0; i < products.length; i++) {
+console.log(products[i].length);
+    
+}
+
+/*create a function called calculate price to check the length of the
+products names and perform the required function */
+
+function calculatePrice(products) {
+for (i=0; i < products.length; i++) {
+    if (products[i].length % 2 == 0) {
+        product_price[i] = Math.pow(product_price[i], y = 2);
```

Comment:
I think you are asked to ` double the price of the product` E.g if the product price is `4`, the answer should be `4 + 4 = 8` or `4 * 2 = 8` not `4^2 = 16`